### ISSUE TITLE
Timers and cooldowns get really fucky with time dilation compared to other things

### ISSUE THREAD

**Issue by Robustitron** [tgstation/tgstation#51164]
## Testmerges:
#50558: '[REVIEW] Adds Medical Wounds: Bamboo Bones and the Skin of Your Teeth' by Ryll-Ryll at commit 0083f7b4cf
#50981: 'Completely disables papercode' by optimumtact at commit c21fde5f17
#51071: 'Midrounds and latejoin rulesets now pre-execute' by kriskog at commit 0f2a994ffa
#51148: 'Fix ghosts alt/ctrl clicking atmos devices' by TetraK1 at commit 830e6e937a

## Reproduction:
-Be on a server with high-ish time dilation (on bagil it usually starts at ~40%
-Try priming a grenade, getting slipped/stunned, or getting stuck in zero g, throwing something, and staying stuck for awhile
-Notice the timers and cooldowns go up way higher than the lag would normally explain, you can be slipped or stuck in zero g for up to a minute before moving again.

Basically the game used to slow down evenly in a sort of linear way with time dilation in the past but now some stuff related to timers and cooldowns seem to break down and go up exponentially to a degree that don't reflect the time dilation percentage.

**Comment by LemonInTheDark** [comment]
Not sure if this is caused by explosion pausing or not, goof is gonna bean me, but I'ma ping him anyway @Iamgoofball

**Comment by Iamgoofball** [comment]
Explosion pausing got removed but this is something to look into.

**Comment by bobbahbrown** [comment]
there are an impressive number of runtimes in this round, and they appear to start from the getgo during round initialization

https://tgstation13.org/parsed-logs/basil/data/logs/2020/05/19/round-137433/runtime.txt

**Comment by kriskog** [comment]
If timer based cooldowns are taking a much longer time to recharge than usual, it is likely due to the timer subsystem going overtime and timers getting queued. Ive seen this a few times, it breaks a lot of stuff. Some times it recovers given enough time, other times it just keeps accumulating more work than it has time to do.

**Comment by Qustinnus** [comment]
we should move things like the chat messages to their own subsystem

**Comment by kriskog** [comment]
While timers and our use of them are probably not optimal, the times I have witnessed this happen atmos and lighting have also been eating all the free time per tick, so there is no extra time to recover from.

**Comment by timothymtorres** [comment]
Linking these all together since they seem possibly related or dupes: 

#56292 
#42741
#54881
#51164
#48484
#30423
